giraffe/src/components/Geo.tsx

```diff
@@ -64,7 +63,19 @@ const getMinZoom = (width: number): number => {
   return Math.ceil(Math.log2(width / 256) * ZOOM_FRACTION) / ZOOM_FRACTION
 }
 
+const loadLeafletStyles = (loaded => {
+  return () => {
+    if (!loaded) {
+      loaded = true
+      require(['leaflet/dist/leaflet.css'], result => {
+        result.default.use()
+      })
+    }
+  }
+})(false)
```

Comment:
why not just load this through a normal import?

Reply:
Optimization that worked for me is that the leaflet css will be embedded into the page only in the case the geo widget is used. It won't be parsed otherwise - it is ~70kb file if I remember correctly. It is better than nothing I think.